docs/nginx-ingress-controller.md

```diff
@@ -153,4 +153,14 @@ spec:
 
 | Field | Type | Description | Required |
 | --- | --- | --- | --- |
-| `enable` | `boolean` | Enable App Protect. | Yes |
+| `enable` | `boolean` | Enable App Protect WAF. | Yes |
+
+## NginxIngressController.AppProtectDos
```

Comment:
Could you add a row about appProtectDos field in the main table and link to this section? This is for consistency, as we document all other second-level fields this way.

Reply:
added